contract/tracking.proto

```diff
@@ -49,6 +49,7 @@ enum Subject {
   SUBJECT_METER_READING = 1;
   SUBJECT_CUSTOMER_REFERRAL = 2;
   SUBJECT_BILL = 3;
+  SUBJECT_PREFERENCES = 4;
```

Comment:
Preferences is a very all encompassing term, I wonder if we could be a bit more specific there?

Reply:
Could go with ENERGY_PREFERENCES I guess that will be easier to drill into from name alone without looking at underlying property.